htmlToElement.js

```diff
@@ -16,24 +17,39 @@ const defaultOpts = {
 };
 
 const Img = props => {
-  const width =
+  let widthAtt =
     parseInt(props.attribs['width'], 10) || parseInt(props.attribs['data-width'], 10) || 0;
   const height =
     parseInt(props.attribs['height'], 10) ||
     parseInt(props.attribs['data-height'], 10) ||
     0;
 
+  const { width } = Dimensions.get('window');
+  const resizeMode = null;
+
+  if (!widthAtt) {
+    widthAtt = (width - 20);
+    resizeMode: 'contain'
+  }
+
   const imgStyle = {
-    width,
-    height,
+    width: widthAtt,
+    maxWidth: (width - 20),
+    height: height || 200,
+    resizeMode
   };
 
   const source = {
     uri: props.attribs.src,
-    width,
+    width: widthAtt,
     height,
   };
-  return <AutoSizedImage source={source} style={imgStyle} />;
+
+  if (props.attribs.src) {
+    return <AutoSizedImage source={source} style={imgStyle} />;
+  } else {
+    return <Text>-</Text>
```

Comment:
I guess so. It feels weird returning a `Text` where the code expects an `Image`. I wonder if it's worth including a tiny static 'x' image to indicate broken images...